src/main/java/ch/puzzle/eft/controller/SiteController.java

```diff
@@ -59,7 +58,7 @@ public String viewValidatePage(@Valid ExamNumberForm examNumberForm, BindingResu
     @PostMapping("/")
     public ResponseEntity<String> acceptCookies(Model model) {
         ResponseCookie.ResponseCookieBuilder cookieBuilder = ResponseCookie.from("cookie-consent", "true");
-        cookieBuilder.maxAge(60 * 60 * 24 * 365);
+        cookieBuilder.maxAge((long) 60 * 60 * 24 * 365);
```

Comment:
This works fine but why not use `java.time.Duration` for this? I think that makes it more readable. 
```suggestion
        cookieBuilder.maxAge(Duration.ofDays(365).toSeconds());
```